az-core/src/main/java/azkaban/utils/Props.java

```diff
@@ -430,6 +431,23 @@ public String getString(final String key) {
   }
 
   /**
+   * Returns a list of clusters with the comma as the separator of the value
+   * e.g., for input string: "thrift://hcat1:port,thrift://hcat2:port;thrift://hcat3:port,thrift://hcat4:port;"
```

Comment:
What if the string has some format error? What will happen? Like 
```
thrift:hcat1:port;thrift:hcat2:port
```

Reply:
In this way, I think hive will throw an exception.